src/SignalR/common/Http.Connections/ref/Microsoft.AspNetCore.Http.Connections.netcoreapp.cs

```diff
@@ -30,7 +30,7 @@ public partial class ConnectionOptions
     }
     public partial class ConnectionOptionsSetup : Microsoft.Extensions.Options.IConfigureOptions<Microsoft.AspNetCore.Http.Connections.ConnectionOptions>
     {
-        public static System.TimeSpan DefaultDisconectTimeout;
+        public static readonly System.TimeSpan DefaultDisconectTimeout;
```

Comment:
This is also a breaking change. In general, anything that modifies code in a `ref/` directory will be a breaking change. I'm a little disappointed this wasn't `readonly` to begin with but again, we need to be cautious here.

That doesn't mean we can't take it! It just means we have to get some consensus among the team about it.

Reply:
Can we obsolete the API? This probably should have been internal to begin with.

Reply:
That's one option. We could just phase it out starting in 5.0.

Reply:
Let's leave this be. We don't think it's worth while obsoleting or changing this.

Reply:
Still needs to be reverted.